### DISCUSSION TITLE
Today is my Birthday! I got ADHD - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#59] [General]
# Today is my Birthday! I got ADHD - Alan Pope's blog

This is a deeply personal post. Feel free to skip this if you’re only here for the Linux and open-source content. It’s also a touch rambling. As for the title, no, I didn’t “get” ADHD on my birthday; obviously, that’s humourous literary hyperbole. Read on.
LET age = age + 1 Like a few billion others, I managed to cling to this precious rock we call home and complete a 52nd orbit of our nearest star.

https://popey.com/blog/2024/04/today-is-my-birthday-i-got-adhd/

**Comment by skellat** [comment]
👍

**Comment by stefanmachura** [comment]
Thank you for this post. As a person who, as you, is suspecting themselves of ADHD and being on the spectrum, your post gave me another incentive to do something with it and reach out for a diagnosis.

**Comment by M2vH** [comment]
Happy Birthday. 👋🎂👍

**Comment by absoludity** [comment]
Quite a journey popey - hope your next four revolutions around this star are better than the last four!

**Comment by phillipeaton** [comment]
Well done for writing the post. My significant other also has a recent diagnosis and seem to bring closure to her in the same way it has you. 🤸‍♂️
